backend/src/config.ts

```diff
@@ -0,0 +1,24 @@
+import {Keypair, PublicKey} from "@solana/web3.js";
+
+const base64ToUint8Array = (privateKey: string | undefined): Uint8Array | undefined => {
+  if (!privateKey) return;
+  return Uint8Array.from(
+    Buffer.from(privateKey, 'base64')
+      .toString('ascii')
+      // Removes all whitespace and newlines
+      .replace(/[\s\n]+/g, '')
+      .split(',')
+      .map((c) => parseInt(c, 10)));
+}
+
+export const NETWORK = new PublicKey(process.env.NETWORK ?? '');
+export const GATEKEEPER = new PublicKey(process.env.GATEKEEPER ?? '');
```

Comment:
This can be derived from the GATEKEEPER_AUTHORITY ? (not necessarily - but if we use the same key that could simplify it?)

Reply:
Nvm, I prefer this tbh as the authority doesn't have to be the same as the gatekeeper account

Reply:
Thanks!